### PR TITLE
Whats new: Add step editor and copy link 

### PR DESCRIPTION
**Pull request by ivanahuckova**
This PR adds following new 10.1 features to What's new docs:

- Loki step editor
- Copy link to a Loki log line

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v10-1.md [hunk 1]
```diff
@@ -59,3 +59,27 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
...
+
+## Data sources
+
+### Step editor in Loki
+
+<!-- Ivana Huckova -->
+
+_Generally available in all editions of Grafana._
+
+We've made an improvement to the Loki query editor by adding a new step editor. This editor allows you to specify a value for the "step" parameter in Loki queries. You can use this parameter when making metric queries to Loki or when you want to get a matrix response from your queries.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-1.md:71]:
```suggestion
We've improved the Loki query editor by adding a new step editor. This editor allows you to specify a value for the _step_ parameter in Loki queries. You can use this parameter when making metric queries to Loki or when you want to get a matrix response from your queries.
```

File: docs/sources/whatsnew/whats-new-in-v10-1.md [hunk 2]
```diff
@@ -59,3 +59,27 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
...
+
+{{< figure src="/media/docs/grafana/data-sources/loki-step-editor.png" max-width="750px" caption="New Loki step editor" >}}
+
+### Copy link to a Loki log line
+
+<!-- Sven Grossmann -->
+
+_Generally available in all editions of Grafana._
+
+A new linking of Loki log lines in Explore allows you to quickly navigate to specific log entries for precise analysis. By clicking the **Copy shortcut** button for a log line, you can generate and copy a [short URL]({{< relref "../developers/http_api/short_url/" >}}) that provides direct access to the exact log entry within an absolute time range. When you open the link, Grafana will automatically scroll to the corresponding log line and highlight it with a blue background, making it easy to identify and focus on the relevant information.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-1.md:83]:
```suggestion
A new linking of Loki log lines in Explore allows you to quickly navigate to specific log entries for precise analysis. By clicking the **Copy shortcut** button for a log line, you can generate and copy a [short URL]({{< relref "../developers/http_api/short_url/" >}}) that provides direct access to the exact log entry within an absolute time range. When you open the link, Grafana automatically scrolls to the corresponding log line and highlights it with a blue background, making it easy to identify and focus on the relevant information.
```

**Review by imatwawana — APPROVED** [review]
Thanks for adding! A couple wording nits. I'll do a more thorough edit after the cutoff. But otherwise LGTM!

File: docs/sources/whatsnew/whats-new-in-v10-1.md [hunk 3]
```diff
@@ -59,3 +59,27 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
...
+
+{{< figure src="/media/docs/grafana/data-sources/loki-step-editor.png" max-width="750px" caption="New Loki step editor" >}}
+
+### Copy link to a Loki log line
+
+<!-- Sven Grossmann -->
+
+_Generally available in all editions of Grafana._
+
+A new linking of Loki log lines in Explore allows you to quickly navigate to specific log entries for precise analysis. By clicking the **Copy shortcut** button for a log line, you can generate and copy a [short URL]({{< relref "../developers/http_api/short_url/" >}}) that provides direct access to the exact log entry within an absolute time range. When you open the link, Grafana automatically scrolls to the corresponding log line and highlights it with a blue background, making it easy to identify and focus on the relevant information.
```
Comment (gabor) [inline docs/sources/whatsnew/whats-new-in-v10-1.md:83]:
```suggestion
A new linking of Loki log lines in Explore allows you to quickly navigate to specific log entries for precise analysis. By clicking the **Copy shortlink** button for a log line, you can generate and copy a [short URL]({{< relref "../developers/http_api/short_url/" >}}) that provides direct access to the exact log entry within an absolute time range. When you open the link, Grafana automatically scrolls to the corresponding log line and highlights it with a blue background, making it easy to identify and focus on the relevant information.
```

the button says "Copy shortlink"

**Pull request merged**